src/js/main.js

```diff
@@ -26,39 +26,48 @@ import downloadFile from './rest/downloadFile';
 import uploadFile from './rest/uploadFile';
 import getFormFields from './rest/getFormFields';
 import getFormLayout from './rest/getFormLayout';
+import postDeploy from './rest/postDeploy';
+import getDeploy from './rest/getDeploy';
+import getCustomize from './rest/getCustomize';
+import putCustomize from './rest/putCustomize';
 
 let kintoneUtility = {
-    rest: {},
+    rest: {
+        getRecord,
+        getRecords,
+        getAllRecordsByQuery,
+        postRecord,
+        postRecords,
+        postAllRecords,
+        putRecord,
+        putRecords,
+        putAllRecords,
+        deleteRecords,
+        deleteAllRecords,
+        deleteAllRecordsByQuery,
+        upsertRecord,
+        upsertRecords,
+        downloadFile,
+        uploadFile,
+        getFormFields,
+        getFormLayout,
+        postDeploy,
```

Comment:
I think the function name should be more clear
+ postDeploy => postDeployAppSettings